cookbooks/aws-parallelcluster-environment/recipes/init/mount_internal_use_efs.rb

```diff
@@ -57,6 +57,15 @@
     end
   end unless initial_shared_dir_array.empty?
 
+  # Add the mount point for home
+  directory "#{node['cluster']['internal_initial_shared_dir']}/home" do
+    user 'root'
+    group 'root'
+    mode '0755'
+    action :create
+    recursive true
+  end
```

Comment:
If `/home` mount is from the `SharedStorage` section, the `/home` directory in this EFS is not used. Do you think it is confusing for users?

Reply:
It could potentially be, but the users would never see it if they don't mount it.  I was trying to avoid remounting the initial shared dir for the shared home section